outdated/modules/httpd.reb

```diff
@@ -112,29 +127,26 @@ sys/make-scheme [
                 spec/port-id: spec/ref/3
                 spec/actions: spec/ref/4
             ]
-            /else [
-                do make error! "Server lacking core features."
-            ]
+            default [fail "Server lacking core features."]
         ]
 
         server/locals: make object! [
-            handler: subport: open?: _
+            handler: _
+            subport: _
+            open?: _
             clients: make block! 1024
         ]
 
-        server/locals/handler: procedure [
+        server/locals/handler: function [
+            return: <void>
             request [object!]
             response [object!]
         ] compose [
             render: get in response 'render
             redirect: get in response 'redirect
             print: get in response 'print
 
-            (
-                block? server/spec/actions
-                    then [server/spec/actions]
```

Comment:
Non-Stylistic change: [THEN isn't based on conditional logic on its left](https://forum.rebol.info/t/taking-then-back-as-the-rightful-complement-to-else/731).   This would have to be changed to an AND to be logic-driven on the left.